src/components/menu/menu.spec.ts

```diff
@@ -536,4 +550,22 @@ describe('MenuTriggerDestroyTestComponent', () => {
         expect(document.querySelectorAll('.ux-menu').length).toBe(1);
     });
 
+    it('should contain the aria-controls attribute when the menu is opened', async () => {
+        expect(document.querySelector('.btn').getAttribute('aria-controls')).toBeFalsy();
+
+        // open menu
+        component.trigger.openMenu();
+
+        fixture.detectChanges();
+        await fixture.whenStable();
+
+        // expect menu to be open
+        expect(document.querySelectorAll('.ux-menu').length).toBe(1);
```

Comment:
I see there's an existing test with this assertion, so we don't need to duplicate it in the new tests.